CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
    - Storage systems
    - Switches
    - Volume
+   - Volume attachment
```

Comment:
Well, this resource should not be added on a tag that was already released, it should be in a new entry like *3.1.0*